package.json

```diff
@@ -101,7 +101,7 @@
     "stylelint": "stylelint --ignore-path .gitignore \"**/*.scss\"",
     "validate": "npm run eslint && npm run stylelint",
     "format": "npm run eslint -- --fix",
-    "test": "npm run validate && lerna run test",
+    "test": "npm run validate && jest && lerna run test:mocha",
```

Comment:
In the subpackages we can't use `test:unit` for mocha because while we're rewriting tests to use jest (see also the other PR about core shell tests) as the runner we need to be able to run both separately. So I'd keep `test:mocha` for the "legacy tests" and keep `test:unit` for new jest tests.

Reply:
Also, is there a point to having the `test:unit` script in subpackages? It feels unnnecessary duplication to me, especially since the goal would be to run (jest) tests only from root to measure coverage correctly. Plus, it will probably be faster to only run jest once from root, instead of starting the process n times in every package.